packages/cli/src/executions/executions.controller.ts

```diff
@@ -59,6 +61,10 @@ export class ExecutionsController {
 
 	@Get('/:id')
 	async getOne(req: ExecutionRequest.GetOne) {
+		if (!isPositiveInteger(req.params.id)) {
```

Comment:
we don't need to make that change now, but stuff like this should use class-validator/transformer to guarantee that `req.params.id` is a valid number by the time the code-execution reaches this function.

Reply:
Yup we need a more general solution, either class-validator or Zod or what Val's RFC ends up with.